### ISSUE TITLE
CICE writing extra restarts

### ISSUE THREAD

**Issue by anton-seaice** [ACCESS-NRI/access-om3-configs#359]
TL;DR: The CICE cmeps driver is not using the restart settings from `nuopc.runconfig` and is using those from `ice_in`

Using _1deg_jra55do_iaf_ I set:

     restart_n = 5
     restart_option = nyears
     stop_n = 5
     stop_option = nyears

in `nuopc.runconfig`

I then ran access-om3 (access-om3-d6813d6b9e1df560ac3f6ba6a605daab9cfd9569_main-5pjh7z2) using payu 1.1.3. The model completed and ran succesfully.

The _archive/output000_ folder incorrectly contained:

    archive/output000/access-om3.cice.r.1959-01-01-00000.nc  archive/output000/access-om3.cice.r.1960-01-01-00000.nc  archive/output000/access-om3.cice.r.1961-01-01-00000.nc  archive/output000/access-om3.cice.r.1962-01-01-00000.nc

The correct restart was saved here:

    archive/restart000/access-om3.cice.r.1963-01-01-00000.nc

The PBS error log also has this error from:

    ./archive_cice_restarts.sh: line 4: [: too many arguments

This is because CICE is writing annual restarts due to these settings (from `ice.log`):

dumpfreq         =      y     x     x     x     x  (set in `ice_in`)
dumpfreq_n       =       1     1     1     1     1 (the default)

I think the desired behaviour in the cmeps driver for CICE ignores/replaces and values from `ice_in` and sets them from the values in `nuopc.runconfig` ?
